openedx/features/caliper_tracking/tests/current/edx.video.stopped.json

```diff
@@ -0,0 +1,21 @@
+{
+  "accept_language": "en-US,en;q=0.5",
```

Comment:
Please rename both JSON files to `stop_video.json`

Reply:
Done (y). Kindly review it again.